pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -151,6 +156,20 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	if checkEngineStatusForStop(engine) {
+		r.reconcileForDelete(request)
```

Comment:
You are returning the `reconcile.Result, error` but here you are not doing anything!!